src/view/uniprotkb/components/protvista-config/variationFilters.ts

```diff
@@ -0,0 +1,215 @@
+      getFilteredVariants(
+        variants,
+        (variantPos: TransformedProtvistaVariant) =>
+          typeof variantPos.polyphenScore !== 'undefined' ||
+          typeof variantPos.siftScore !== 'undefined'
+      ),
+  },
+  {
+    name: 'nonDisease',
+    type: {
+      name: 'consequence',
+      text: 'Filter Consequence',
+    },
+    options: {
+      labels: ['Likely benign'],
+      colors: [scaleColors.UPNonDiseaseColor],
+    },
+    filterData: (variants: VariationVariants) =>
+      getFilteredVariants(variants, (variantPos: TransformedProtvistaVariant) =>
+        significanceMatches(
+          consequences.likelyBenign,
+          variantPos.clinicalSignificances
+        )
+      ),
+  },
+  {
+    name: 'uncertain',
+    type: {
+      name: 'consequence',
+      text: 'Filter Consequence',
+    },
+    options: {
+      labels: ['Uncertain'],
+      colors: [scaleColors.othersColor],
+    },
+    filterData: (variants: VariationVariants) =>
+      getFilteredVariants(
+        variants,
+        (variantPos: TransformedProtvistaVariant) =>
+          (typeof variantPos.clinicalSignificances === 'undefined' &&
+            typeof variantPos.polyphenScore === 'undefined' &&
+            typeof variantPos.siftScore === 'undefined') ||
+          significanceMatches(
+            consequences.uncertain,
+            variantPos.clinicalSignificances
+          )
+      ),
+  },
+  {
+    name: 'UniProt',
+    type: {
+      name: 'provenance',
+      text: 'Filter Provenance',
+    },
+    options: {
+      labels: ['UniProt reviewed'],
+      colors: ['#9f9f9f'],
+    },
+    filterData: (variants: VariationVariants) =>
+      getFilteredVariants(
+        variants,
+        (variantPos: TransformedProtvistaVariant) =>
+          variantPos.xrefNames &&
+          (variantPos.xrefNames.includes('uniprot') ||
+            variantPos.xrefNames.includes('UniProt'))
+      ),
+  },
+  {
+    name: 'ClinVar',
+    type: {
+      name: 'provenance',
+      text: 'Filter Provenance',
+    },
+    options: {
+      labels: ['ClinVar reviewed'],
+      colors: ['#9f9f9f'],
+    },
+    filterData: (variants: VariationVariants) =>
+      getFilteredVariants(
+        variants,
+        (variantPos: TransformedProtvistaVariant) =>
+          variantPos.xrefNames &&
+          (variantPos.xrefNames.includes('ClinVar') ||
+            variantPos.xrefNames.includes('clinvar'))
+      ),
+  },
+  {
+    name: 'LSS',
+    type: {
+      name: 'provenance',
+      text: 'Filter Provenance',
+    },
+    options: {
+      labels: ['Large scale studies'],
+      colors: ['#9f9f9f'],
+    },
+    filterData: (variants: VariationVariants) =>
+      getFilteredVariants(
+        variants,
+        (variantPos: TransformedProtvistaVariant) =>
+          variantPos.sourceType === 'large_scale_study' ||
+          variantPos.sourceType === 'mixed'
+      ),
+  },
+];
+
+const predictionScale = scaleLinear<string>()
```

Comment:
This is more observation and speculation: I am wondering if a non-linear scale is needed here as it seems the thresholds happens at odd places in the range [0,1]?
```
PolyPhen
0.0 to 0.15 -- Variants with scores in this range are predicted to be benign.
0.15 to 1.0 -- Variants with scores in this range are possibly damaging.
0.85 to 1.0 -- Variants with scores in this range are more confidently predicted to be damaging.

SIFT
0.0 to 0.05 -- Variants with scores in this range are considered deleterious. Variants with scores closer to 0.0 are more confidently predicted to be deleterious.
0.05 to 1.0-- Variants with scores in this range are predicted to be tolerated (benign). Variants with scores very close to 1.0 are more confidently predicted to be tolerated.
```

Reply:
It gets complicated as what we show is either, or, or a product of both sift and polyphen, and the scales are different. 
Initially we had planned on doing something more clever, but in the end because we are not using discrete prediction values but a colour scale (or luminance I think?) it is impossible for someone to know exactly under what category the variant falls by simply looking at the colour (and rightly so, as these are just indications and not very accurate predictions...) so the idea was to provide just an indication of the potential severity of the mutation. So we're staying deliberately vague.
There are also new prediction methods that Andrew is working on importing, and some are more accurate so we can review this then.